models/calendar.yml

```diff
@@ -185,7 +185,17 @@
         title: Introductions
         emoji: 👋
 
-    - 7:45 PM: Speaker TBA
+    - 7:45 PM:
+        type: Talk
+        emoji: 🐛🏳‍🌈
+        title: |
+          "Works for Me": Understanding Privilege in Tech
+        description: |
+          "Works for me." The words spoken to dismiss a thousand bugs. "Can'tReproduce." Issue closed. "You're getting worked up over nothing." "It was just a joke." "I worked just as hard to get here." "It didn't bother me." Problem dismissed. "Works for me" is a natural reaction when confronted with cultural issues that don't impact us directly. It's a statement that isn't inaccurate, but is wildly incomplete. We have to learn to see past our own experiences and see and accept the experiences of others. This talk will dive into how and why dismissal of culture concerns happens in the tech community, and how we can combat it with knowledge and empathy.
```

Comment:
Nice catch! I guess it was a copy-paste error or something on my part. Today really is a Monday for me 😅